settings.php

```diff
@@ -139,8 +139,14 @@
     $title = get_string('avatarnumberstorenders', 'hsuforum');
     $description = get_string('avatarnumberstorendersdescription', 'hsuforum');
     $default = '';
-    $setting = new admin_setting_configtext($name, $title, $description, $default);
-    $settings->add($setting);
+    $options = array(
+        ''  => 'default',
```

Comment:
What would the impact be of having an empty value. How many participant icons will show?

Reply:
will default to all the participants

Reply:
Please confirm with the BSA that this is the expected behavior. From an engineering perspective there might be i.e 20 participants and showing all will bloat that block up in a negative way.

Reply:
@kurvinh mentioned this will be expected behavior after checking in with bsa